bundles/vi/base_map.moon

```diff
@@ -29,16 +29,18 @@ back_to_char = (event, source, translations, editor) ->
 forward_till_char = (event, source, translations, editor) ->
   if event.character
     apply editor, (editor) ->
+      current_pos = editor.cursor.pos
       editor\forward_to_match event.character
-      editor.cursor\left!
+      if current_pos != editor.cursor.pos then editor.cursor\left!
```

Comment:
Style nitpick: We don't typically do `then` like that, so either move the action clause below on it's own line or go with a statement modifier, e.g.:

`editor.cursor\left! unless current_pos == editor.cursor.pos`

The first alternative would probably look better here. Ditto for the change below.